### PR TITLE
New Haven: enable ti.to and registrations sections

### PR DESCRIPTION
**Pull request by martymcguire**

### CODE REVIEW

File: public/newhaven.php [hunk 1]
```diff
@@ -112,27 +112,27 @@
   <div class="ui vertical stripe segment" id="register">
     <div class="ui text container">
       <h3 class="ui header">Registration TBA</h3>
-<!--      <tito-widget event="indiewebcamp/baltimore-2018"><a href="https://ti.to/indiewebcamp/baltimore-2018">Get Tickets</a></tito-widget>-->
+      <tito-widget event="indiewebcamp/baltimore-2018"><a href="https://ti.to/indiewebcamp/baltimore-2018">Get Tickets</a></tito-widget>
```
Comment (EdwardHinkle) [inline public/newhaven.php]: This is still pointing at Baltimore

File: public/newhaven.php [hunk 2]
```diff
@@ -112,27 +112,27 @@
...
-<!--    <div class="ui text container">-->
-<!--      <h3 class="ui header">Indie RSVPs</h3>-->
+  <div class="ui vertical stripe segment" id="rsvps">
+    <div class="ui text container">
+      <h3 class="ui header">Indie RSVPs</h3>
 
-<!--      <p>See <a href="https://indieweb.org/RSVP">indieweb.org/RSVP</a> for instructions on how to create an RSVP post. Once you've created the RSVP post which links to this page, send a Webmention and you'll appear below!</p>-->
+      <p>See <a href="https://indieweb.org/RSVP">indieweb.org/RSVP</a> for instructions on how to create an RSVP post. Once you've created the RSVP post which links to this page, send a Webmention and you'll appear below!</p>
 
         <?php //include('../templates/show-rsvps.php'); ?>
```
Comment (EdwardHinkle) [inline public/newhaven.php]: Should probably uncomment this to show RSVPs via Webmention

**Review by EdwardHinkle — CHANGES_REQUESTED** [review]
A couple issues

File: public/newhaven.php (hunk 1 above)
Comment (martymcguire) [reply public/newhaven.php]: updated to `indiewebcamp/new-haven`

File: public/newhaven.php (hunk 2 above)
Comment (martymcguire) [reply public/newhaven.php]: uncommented! also uncommented display of ti.to RSVPs.

**Review by EdwardHinkle — APPROVED** [review]

**Pull request merged**